### PR TITLE
Pharma - Disable Painkiller stacking

### PR DESCRIPTION
**Pull request by Cplhardcore**
**When merged this pull request will:**
Add the ability to prevent painkillers pain suppression from stacking, requiring the need for higher level painkillers to be used
- _Describe what this pull request will do_
It adds a CBA setting to enable the stacking effect or disable it along with edits to fnc_treatmentAdvanced_Painkillers.sqf 
- _Each change in a separate line_

Will need localization

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/pharma/functions/fnc_treatmentAdvanced_Painkillers.sqf [hunk 1]
```diff
@@ -21,9 +21,10 @@
  */
 
 params ["_medic", "_patient", "_bodyPart", "_classname", "", "_usedItem"];
+private _count1 = [_patient, "Painkillers"] call ACEFUNC(medical_status,getMedicationCount);
```
Comment (MiszczuZPolski) [inline addons/pharma/functions/fnc_treatmentAdvanced_Painkillers.sqf]: i would rename it personally, count1 is not good name

Comment (Cplhardcore) [reply addons/pharma/functions/fnc_treatmentAdvanced_Painkillers.sqf]: Renamed to painkillerscount

**Comment by Cplhardcore** [comment]
Small issue i realized, if you take a painkiller while you have the effects of a previous dose in your system, the pain suppression will run out and not be replenished

**Comment by Cplhardcore** [comment]
Closing this as there is no current way to add time onto an already active pain suppression source without it stacking

**Pull request closed without merging**